code/game/turfs/simulated/floor/lava.dm

```diff
@@ -103,6 +103,13 @@
 
 			if("lava" in L.weather_immunities)
 				continue
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				var/obj/item/hat =H.head
```

Comment:
spacing

Reply:
the best bad spacer this is. Should be fixed now